i18n breaks usage outside of jsonschema2md repo

Hi @Claymore1337 it looks like the latest release is broken when used from `node_modules` of another project:

```bash
$ npx jsonschema2md -d src/schemas -o docs

07
info: output directory: /home/circleci/project/docs
info: finished reading all *.schema.json files in /home/circleci/project/src/schemas, beginning processing….
/home/circleci/project/src/schemas/action.schema.json
/home/circleci/project/src/schemas/content.schema.json
/home/circleci/project/src/schemas/context.schema.json
/home/circleci/project/src/schemas/mdast.schema.json
/home/circleci/project/src/schemas/meta.schema.json
/home/circleci/project/src/schemas/position.schema.json
/home/circleci/project/src/schemas/rawrequest.schema.json
/home/circleci/project/src/schemas/request.schema.json
/home/circleci/project/src/schemas/response.schema.json
/home/circleci/project/src/schemas/secrets.schema.json
/home/circleci/project/src/schemas/section.schema.json
unresolved reference: https://ns.adobe.com/helix/pipeline/position
unresolved reference: https://ns.adobe.com/helix/pipeline/mdast
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta/properties/title
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta/properties/intro
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta/properties/image
unresolved reference: https://ns.adobe.com/helix/pipeline/mdast
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta/properties/image
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta/properties/intro
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta
unresolved reference: https://ns.adobe.com/helix/pipeline/position
unresolved reference: https://ns.adobe.com/helix/pipeline/meta#/definitions/meta/properties/title
/home/circleci/project/src/schemas/textcoordinates.schema.json
Output processed. Trying to make a README.md now
error: ENOENT: no such file or directory, mkdir '/home/circleci/project/lib/locales' {"errno":-2,"code":"ENOENT","syscall":"mkdir","path":"/home/circleci/project/lib/locales"}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @adobe/helix-pipeline@4.0.5 docs: `npx jsonschema2md -d src/schemas -o docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @adobe/helix-pipeline@4.0.5 docs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2019-07-30T09_41_43_235Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @adobe/helix-pipeline@4.0.5 types: `npm run docs && node bin/schema2ts.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @adobe/helix-pipeline@4.0.5 types script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2019-07-30T09_41_43_247Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @adobe/helix-pipeline@4.0.5 lint: `npx eslint . && npm run types`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @adobe/helix-pipeline@4.0.5 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2019-07-30T09_41_43_257Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @adobe/helix-pipeline@4.0.5 test-ci: `npm run lint && npm run test && codecov`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @adobe/helix-pipeline@4.0.5 test-ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2019-07-30T09_41_43_267Z-debug.log
Exited with code 1
```

See also https://github.com/adobe/helix-pipeline/issues/432

--- a/cli.js
+++ b/cli.js
@@ -54,6 +54,7 @@ var argv = require('optimist')
   .argv;
 
 const docs = _.fromPairs(_.toPairs(argv).filter(([ key, value ]) => { return key.startsWith('link-'); }).map(([ key, value ]) => { return [ key.substr(5), value ];}));
+const i18n = require('i18n');
 
 const logger = winston.createLogger({
   level: 'info',
@@ -82,7 +83,6 @@ var schemaDir = argv.x === '-' ? '' : argv.x ? path.resolve(argv.x) : outDir;
 var target = fs.statSync(schemaPath);
 const readme = argv.n !== true;
 const schemaExtension = argv.e || 'schema.json';
-
 if (argv.s){
   ajv.addMetaSchema(require(path.resolve(argv.s)));
 }
@@ -102,6 +102,19 @@ if (argv.m) {
     }
   }
 }
+let i18nPath;
+if (argv !== undefined && argv.i !== undefined){
+  i18nPath=path.resolve(argv.i) ;
+} else {
+  i18nPath=path.resolve(path.join(__dirname, 'lib/locales'));
+}
+i18n.configure({
+  // setup some locales - other locales default to en silently
+  locales:[ 'en' ],
+  // where to store json files - defaults to './locales' relative to modules directory
+  directory: i18nPath,
+  defaultLocale: 'en'
+});
 
 logger.info('output directory: %s', outDir);
 if (target.isDirectory()) {

--- a/lib/markdownWriter.js
+++ b/lib/markdownWriter.js
@@ -171,20 +171,7 @@ const generateMarkdown = function(filename, schema, schemaPath, outDir, dependen
   };
 
   outDir = outDir ? outDir : path.resolve(path.join('.', 'out'));
-  let i18nPath;
-  if (consoleArgs !== undefined && consoleArgs.i !== undefined){
-    i18nPath=path.resolve(consoleArgs.i) ;
-  } else {
-    i18nPath=path.resolve(path.join(__dirname, 'locales'));
-  }
   console.log(filename);
-  i18n.configure({
-    // setup some locales - other locales default to en silently
-    locales:[ 'en' ],
-    // where to store json files - defaults to './locales' relative to modules directory
-    directory: i18nPath,
-    defaultLocale: 'en'
-  });
   //console.log(dependencyMap);
   // this structure allows us to have separate templates for each element. Instead of having
   // one huge template, each block can be built individually

--- a/lib/markdownWriter.js
+++ b/lib/markdownWriter.js
@@ -173,9 +173,9 @@ const generateMarkdown = function(filename, schema, schemaPath, outDir, dependen
   outDir = outDir ? outDir : path.resolve(path.join('.', 'out'));
   let i18nPath;
   if (consoleArgs !== undefined && consoleArgs.i !== undefined){
-    i18nPath=consoleArgs.i ;
+    i18nPath=path.resolve(consoleArgs.i) ;
   } else {
-    i18nPath=__dirname + '/locales';
+    i18nPath=path.resolve(path.join(__dirname, 'locales'));
   }
   console.log(filename);
   i18n.configure({

--- a/lib/readmeWriter.js
+++ b/lib/readmeWriter.js
@@ -35,14 +35,6 @@ function directory(full, base) {
  * @param {string} base - schema base directory
  */
 const generateReadme = function(paths, schemas, out, base) {
-  let i18nPath=path.resolve('./lib/locales');
-  i18n.configure({
-    // setup some locales - other locales default to en silently
-    locales:[ 'en' ],
-    // where to store json files - defaults to './locales' relative to modules directory
-    directory: i18nPath,
-    defaultLocale: 'en'
-  });
   const coreinfo = _.values(schemas).map(schema => {
     return {
       id: schema.jsonSchema.$id,

--- a/spec/lib/header.spec.js
+++ b/spec/lib/header.spec.js
@@ -7,7 +7,7 @@ beforeEach(function() {
     colors: true,
     inline: true
   }));
-  let i18nPath=path.resolve('./lib/locales');
+  let i18nPath=path.resolve(path.join(__dirname, '../../lib/locales'));
   i18n.configure({
     // setup some locales - other locales default to en silently
     locales:[ 'en' ],

